joomla/joomla-cms
# [com_contact] Improve search tooltip

#### Summary of Changes

Simple PR to improve the search tooltip and element label (accessibility) in com_contact.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Components -> Contacts -> Contacts , check the tooltip in the search field.
